database/neo4j/neo4j.go

```diff
@@ -71,14 +73,17 @@ func (n *Neo4j) Open(url string) (database.Driver, error) {
 	authToken := neo4j.BasicAuth(uri.User.Username(), password, "")
 	uri.User = nil
 	uri.Scheme = "bolt"
-	multi := uri.Query().Get("x-multi-statement")
+	multi, err := strconv.ParseBool(uri.Query().Get("x-multi-statement"))
```

Comment:
Need to correctly handle `x-multi-statement` not being specified. e.g. an empty string shouldn't error